src/ingest-pipeline/schemata/export_and_backup_map_schema.yml

```diff
@@ -0,0 +1,25 @@
+'$schema': 'http://json-schema.org/schema#'
+'$id': 'http://schemata.hubmapconsortium.org/export_and_backup.yml'
+'title': 'export_and_backup schema'
+'description': 'export_and_backup schema'
+
+'allOf': [{'$ref': '#/definitions/export_and_backup_map'}]
+
+'definitions':
+
+  'export_and_backup_record':
+     'type': 'object'
+     'properties':
+        'status':
+           'type': 'string'
+           'description': 'the status of the dataset or upload'
+        'plugins':
+            'type': 'array'
+            'description': 'array of appropriate plugins for status'
```

Comment:
I think add the following to specify the type of items in the plugins array:
'items': {'type': 'string'}

Reply:
Done!